lib/index.d.ts

```diff
@@ -131,6 +131,7 @@ export function decryptSessionKey(options: {
 
 export interface DecryptResultPmcrypto extends DecryptResult {
     verified: VERIFICATION_STATUS;
+    signature: OpenPGPSignature;
 }
```

Comment:
`decryptMessage` already returns a `signatures` property, each element of which has a `keyid` - this is already specified by `DecryptResult`.
It seems like `decryptMessageLegacy` doesn't conform to this though, it returns `signature: 0` which should be `verified: 0, signatures: []`.